cmd/evm/internal/t8ntool/flags.go

```diff
@@ -79,8 +79,10 @@ var (
 		Value: "env.json",
 	}
 	InputTxsFlag = cli.StringFlag{
-		Name:  "input.txs",
-		Usage: "`stdin` or file name of where to find the transactions to apply.",
+		Name: "input.txs",
+		Usage: "`stdin` or file name of where to find the transactions to apply. " +
```

Comment:
Maybe we can add a comment that RLP list over stdin is not supported
```
echo signed_txs.rlp | ./evm t8n --state.fork=London --input.alloc=./testdata/13/alloc.json --input.txs=stdin --input.env=./testdata/13/env.json --output.result=alloc_rlptx.json
ERROR(10): failed unmarshaling stdin: invalid character 's' looking for beginning of value
```

Reply:
Heh. Try `cat` instead of `echo` :)

Reply:
I'm doofus, but the general criticism still stands. We can't read RLP from stdin, because L147 tries to decode into an input object
```
cat signed_txs.rlp | ./evm t8n --state.fork=London --input.alloc=./testdata/13/alloc.json --input.txs=stdin --input.env=./testdata/13/env.json --output.result=alloc_rlptx.json
ERROR(10): failed unmarshaling stdin: json: cannot unmarshal string into Go value of type t8ntool.input
```

Reply:
Yup, good catch! Fixing...

Reply:
Fixed this in my last commit. One can't just take the output and feed to the input, since the input can contain many things. Therefore, you need to envelop the leaf json into an object, and call it `txsRlp`:
```
[user@work evm]$ cat signed_txs.rlp.json| jq . 
{
  "txsRlp": "0xf8d2b86702f864010180820fa08284d09411111111111111111111111111111111111111118080c001a0b7dfab36232379bb3d1497a4f91c1966b1f932eae3ade107bf5d723b9cb474e0a06261c359a10f2132f126d250485b90cf20f30340801244a08ef6142ab33d1904b86702f864010280820fa08284d09411111111111111111111111111111111111111118080c080a0d4ec563b6568cd42d998fc4134b36933c6568d01533b5adf08769270243c6c7fa072bf7c21eac6bbeae5143371eef26d5e279637f3bd73482b55979d76d935b1e9"
}
[user@work evm]$ cat signed_txs.rlp.json| ./evm t8n --state.fork=London --input.alloc=./testdata/13/alloc.json --input.txs=stdin --input.env=./testdata/13/env.json --output.result=alloc_rlptx.json
INFO [07-28|09:56:08.806] Trie dumping started                     root=672261..31e1e5
INFO [07-28|09:56:08.806] Trie dumping complete                    accounts=3 elapsed="211.905µs"
INFO [07-28|09:56:08.806] Wrote file                               file=alloc.json
INFO [07-28|09:56:08.810] Wrote file                               file=alloc_rlptx.json
```